.semgrep/tests/sol-rules.t.sol

```diff
@@ -581,3 +581,90 @@ contract SemgrepTest__sol_style_enforce_require_msg {
         require(cond);
     }
 }
+
+/// NOTE: The order here is important, because the proxied natspec identifier is used below, it is seen as present for
```

Comment:
Let's add separate files for these rules just to avoid creating bugs for other checks. 

So like sol-safety-use-disable-initializers.t.sol